src/components/Footer/Footer.js

```diff
@@ -327,66 +310,28 @@ const styles = StyleSheet.create({
     marginLeft: 4,
     fontWeight: '500'
   },
-  leftGroupContainer: {
-    flexDirection: 'row',
-    alignItems: 'flex-start'
-  },
-  rightGroupContainer: {
-    flexDirection: 'row',
-    alignItems: 'center'
-  },
-  rowSpaceBetween: {
-    flexDirection: 'row',
-    alignItems: 'center'
-  },
   text: {
     ...FONTS.body3,
-    color: COLORS.gray400
+    color: COLORS.grey410
   },
   vote: (colorBasedCount) => ({
     ...FONTS.body3,
     textAlign: 'center',
-    width: 26,
+    marginHorizontal: 10,
+    minWidth: 28,
     color: colorBasedCount
   }),
-  btn: {
-    height: '100%',
-    justifyContent: 'center',
-    alignItems: 'center',
-    display: 'flex'
-  },
   btnShare: {
     height: '100%',
-    justifyContent: 'center',
-    paddingRight: 13.5,
-    paddingLeft: 18
+    justifyContent: 'center'
   },
   btnComment: (commentValue) => ({
-    height: '100%',
-    justifyContent: 'center',
-    marginLeft: 0,
-    paddingRight: commentValue > 9 ? 10 : 4
+    marginRight: commentValue > 9 ? 10 : 6
   }),
   btnBlock: {
     height: '100%',
     justifyContent: 'center',
     paddingHorizontal: 10
-  },
-  btnDownvote: {
-    height: '100%',
-    justifyContent: 'center',
-    paddingHorizontal: 11
-  },
-  btnUpvote: {
-    height: '100%',
-    justifyContent: 'center',
-    paddingLeft: 11,
-    paddingRight: 22
-  },
-  totalCommentContainer: {
-    flex: 1,
-    height: '100%',
-    justifyContent: 'center'
   }
 });
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [290-334]

The styles for the Footer component have been significantly updated to match the new layout. The `container` style now accepts a parameter `isShortText` to conditionally apply styles. Other styles have also been updated or removed to match the new design. Make sure these changes are consistent across different screen sizes and devices.
```diff
- container: {
-    alignItems: 'center',
-   borderBottomWidth: 1,
-   borderBottomColor: COLORS.gray200
- },
+ container: (isShortText) => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    height: 48,
+    paddingHorizontal: 16,
+    paddingVertical: 10,
+    backgroundColor: COLORS.almostBlack,
+    borderLeftWidth: 1,
+    borderRightWidth: 1,
+    borderColor: COLORS.darkGray,
+    marginTop: isShortText ? -1 : 0
+  }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->